docs/changes.rst

```diff
@@ -23,6 +23,14 @@ Version 0.6 (unreleased)
 ========================
 
 
+Implemented RFCs
```

Comment:
given that we intend to backport it, shouldn't this be in 0.5.1 section?

Reply:
moved